components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/dashboard/HeartBeatComponent.java

```diff
@@ -76,6 +76,7 @@ private HeartBeatComponent(){
     private static final String CHANGE_NOTIFICATION = "changeNotification";
     private static final String DEPLOYED_ARTIFACTS = "deployedArtifacts";
     private static final String UNDEPLOYED_ARTIFACTS = "undeployedArtifacts";
+    private static final String TRACING_CHANGED_ARTIFACTS = "tracingChangedArtifacts";
```

Comment:
Shall we call it `TRACING_CHANGE_DETECTED_ARTIFACTS`?

Reply:
Or else, we can give this a common name and reuse it for all the artifacts that we will support update in the future? (Eg: Endpoint activation/suspension?)

Reply:
This is changed to `STATE_CHANGED_ARTIFACTS`